### DISCUSSION TITLE
No Stats

### DISCUSSION THREAD

**Discussion by portboy** [AdguardTeam/AdGuardHome#6171] [Q&A]
I'm running AG Home (v0.107.36) on an unraid server in a docker container within a mcvlan.
AG is working and blocks all the advertisings.
But I get no statistics.
If I do a nslookup like "nslookup <url> <ip Andguard Home> I can see this request in the statistics.

Bug or missconfig from my side?

**Comment by AaronIsMyBrother** [comment]
I would like to ask you a question. Did you try to start AG with not  'root' user?

**Comment by portboy** [comment]
Within Unraid, docker ist running always as root.
Within the container, I setup a user. But no changhes ....

I enabled debug ... but I can only see my nslookup requests .. all other requests are "invisible"

**Reply by AaronIsMyBrother** [reply]
Thank you. I installed AG using the root user, and then I created an Aaron user. I gave Aaron the ownership of the AG file, but I still couldn't start it using Aaron. I cannot understand and cannot find the startup log.

**Reply by portboy** [reply]
I think ur problem ist not my 🙄

**Reply by iam-py-test** [reply]
Ignore [AaronIsMyBrother](https://github.com/AaronIsMyBrother), they have been spamming random threads.
